### DISCUSSION TITLE
Problema ao adicionar o pacote Microsoft.Data.SqlClient

### DISCUSSION THREAD

**Discussion by willianjuliate** [balta-io/forum#35] [Tira Dúvidas] [answered]
Não estou conseguindo adicionar o pacote Microsoft.Data.SqlClient
![Captura de tela 2022-10-01 183906](https://user-images.githubusercontent.com/16016929/193429366-b8985872-1297-47d5-97a8-19bfb8220baf.png)
![Captura de tela 2022-10-01 184104](https://user-images.githubusercontent.com/16016929/193429367-4fce7c84-525e-43f3-a0f9-8dc9185e1356.png)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @willianjuliate, como vai?

Eu tive este problema temporariamente e também não sei dizer o que ocorria. Infelizmente durante alguns dias, meu nuget via linha de comando dava este mesmo conflito... acredito que seja algo no cache local.

De qualquer forma, o que fiz para resolver foi:
Alternativa 1 - Instalar o pacote pelo Visual Studio ou pelo Rider
Alternativa 2 - Limpar o cache do Nugget
```
nuget locals all -list
nuget locals all -clear
```

**Reply by willianjuliate** [reply]
Fiz uma formatação no meu computador de Win10 para o Win11 deu certo kkkk!

**Reply by andrebaltieri** [reply]
Eita hahahahaha

**Comment by oimarcella** [comment]
Boa noite!
Eu estou tentando fazer a instalação via Visual Studio e é informado que existe uma alta vulnerabilidade, será que ainda é uma boa ideia instalar essa mesma versão 2.1.3?
Gostaria de instalar essa para não ter problemas com versões agora no início do aprendizado.

![image](https://github.com/balta-io/forum/assets/43733159/ef24206c-dc6a-4af9-ad12-9654ba18cf7e)

**Reply by andrebaltieri** [reply]
Pode instalar esta versão sim, só na hora que for para produção é melhor atualizar. Mas para estudos, tudo bem!
